plinth/modules/ejabberd/__init__.py

```diff
@@ -59,6 +60,9 @@
         box_name=_(cfg.box_name))
 ]
 
+web_clients = [web_client(name='JSXC', url='/plinth/apps/jsxc'),
```

Comment:
Keyword arguments aren't always required for namedtuples. Positional parameters work too. It just becomes too verbose sometimes.